demos/python_demos/whiteboard_inpainting_demo/whiteboard_inpainting_demo.py

```diff
@@ -87,14 +90,20 @@ def main():
                         help='Optional. List of monitors to show initially')
     args = parser.parse_args()
 
-    capture = VideoCapture(args.i)
+    cap = open_images_capture(args.input, args.loop)
+    if cap.get_type() not in ('VIDEO', 'CAMERA'):
+        raise RuntimeError("The input should be a video file or a device node")
+    frame = cap.read()
+    if frame is None:
+        raise RuntimeError("Can't read an image from the input")
 
     if bool(args.m_instance_segmentation) == bool(args.m_semantic_segmentation):
         raise ValueError('Set up exactly one of segmentation models: '\
                          '--m_instance_segmentation or --m_semantic_segmentation')
 
-    frame_size, fps = capture.get_source_parameters()
-    out_frame_size = (int(frame_size[0]), int(frame_size[1] * 2))
+    frame_size = frame.shape
+    fps = cap.fps()
+    out_frame_size = (frame_size[1], frame_size[0] * 2)
```

Comment:
It seems weird that `frame_size` and `out_frame_size` have their components in the opposite order. I think they should be consistent.

Reply:
In the previous version the `out_frame_size` stored the tuple consisting of (width, height), that is the opposite of input frame size, so I followed the same logic. Also, the output uses the original `frame_size`: 
```
output_frame = np.full((frame_size[0], frame_size[1], 3), 255, dtype='uint8')
```

Reply:
Maybe to reduce confusion, you could at least remove the `frame_size` variable and use `frame.shape` directly? At least then the names aren't similar.

Reply:
Done.